.ci/azure-pipelines.yml

```diff
@@ -0,0 +1,78 @@
+trigger:
+  batch: true
+  branches:
+    include:
+      - "*"
+  tags:
+    include:
+      - "*"
+pr:
+  branches:
+    include:
+      - "*"
+
+jobs:
+  - job: Build
+    displayName: "Build"
+
+    pool:
+      vmImage: "ubuntu-latest"
+
+    steps:
+      - task: NodeTool@0
+        displayName: "Install node"
+        inputs:
+          versionSpec: "12.x"
+
+      - task: Cache@2
+        displayName: "Check cache"
+        inputs:
+          key: "yarn | yarn.lock"
+          path: "node_modules"
+          cacheHitVar: CACHE_RESTORED
+
+      - script: "apt-get install -y rpmbuild"
+        displayName: "Install apt dependencies"
+
+      - script: "yarn install --frozen-lockfile --non-interactive"
+        displayName: "Install node dependencies"
+
+      - script: "yarn build:linux"
+        displayName: "Build linux packages"
+
+      - script: "mkdir jellyfin-desktop"
+        displayName: "Make output directory"
+
+      - script: "mv dist/*.{AppImage,tar.xz,snap,deb,rpm} jellyfin-desktop"
```

Comment:
Are there other artifacts in the `dist` directory? You might be able to just move it rather than create a new one and move the files individually.

Reply:
Also, if you want to publish releases based on tags you can copy the publish job from Android as well.

Reply:
Yes there are other artifacts that shouldn't be published in the `dist` directory

Reply:
Publish step added